src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5948,4 +5948,16 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_ImplicitRangeIndexerWithName" xml:space="preserve">
     <value>Invocation of implicit Range Indexer cannot name the argument.</value>
   </data>
+  <data name="ERR_TypelessNewIllegalTargetType" xml:space="preserve">
+    <value>The type '{0}' may not be used as the target type of new()</value>
+  </data>
+  <data name="ERR_TypelessNewNotValid" xml:space="preserve">
+    <value>Use of new() is not valid in this context</value>
+  </data>
+  <data name="ERR_TypelessNewNoTargetType" xml:space="preserve">
+    <value>There is no target type for new()</value>
```

Comment:
This should use the `Display` property to show the entire argument list.

Reply:
Since we have no type, we should build the argument list manually, much like what we have for tuples. Is that what you're suggesting?

Reply:
I just want to see the text the user typed between the parens if it exists.

Reply:
For tuples we fetch and join all Display strings from tuples elements. I think we should do the same here as well. Please let me know if there's another way.

Reply:
If that's what we do there, that seems fine to me.